locales/ru_RU.json

```diff
@@ -497,14 +497,14 @@
           "label": "Страна проживания"
         },
         "state": {
-          "label": "Страна",
+          "label": "Штат",
           "location": "Location"
         },
         "confirm-country": {
           "label": "Я подтверждаю, что действительно проживаю в этой стране."
         },
         "confirm-country-and-state": {
-          "label": "Я подтверждаю, что действительно проживаю в этом государстве."
+          "label": "Я подтверждаю, что действительно проживаю в этом штате."
```

Comment:
@bittewenden Please check the EN file. It should be "in this country and state".

```
        "confirm-country-and-state": {
          "label": "I confirm that these are my country and state of residence."
```